main.go

```diff
@@ -84,6 +104,15 @@ func runCmdFunc(cmd *cobra.Command, args []string) error {
 	flags.pretty, _ = cmd.Flags().GetBool("pretty-output")
 	flags.monochrome, _ = cmd.Flags().GetBool("monochrome-output")
 	flags.slurp, _ = cmd.Flags().GetBool("slurp")
+	flags.provideNull, _ = cmd.Flags().GetBool("null-input")
+
+	for _, arg := range stringPositionalArgsFlag.AsSlice() {
+		flags.args = append(flags.args, string(arg))
+	}
+	flags.jsonargs = jsonPositionalArgsFlag.AsSlice()
+	flags.kwargs = stringKwargsFlag.AsMap()
+	flags.jsonkwargs = jsonKwargsFlag.AsMap()
```

Comment:
We should probably refactor this into a constructor for `flags`

Reply:
I'm not 100% sure what you're asking for here.

Reply:
I believe we decided we could refactor flags stuff in another PR.